builder/picolibc.go

```diff
@@ -337,6 +337,7 @@ var picolibcSourcesLarge = []string{
 	"libm/common/math_err_may_uflow.c",
 	"libm/common/math_err_check_uflow.c",
 	"libm/common/math_err_check_oflow.c",
+	"libm/common/math_errf_invalidf.c",
```

Comment:
We might want to add all the other `math_err_*.c` files here too.

Reply:
OK doing it now.

Reply:
Done.